terms/conceptdrift_mf.tex

```diff
@@ -0,0 +1,53 @@
+\chapter[Concept Drift]{Concept Drift}
+\label{ch:concept-drift}\index{Concept Drift|(}
+
+   Nowadays, huge amounts of data are being generated every day. Such data is providing insights on patterns and its importance is increasing over time, thus making it impossible to collect and process data manually. Moreover, simple data analysis is not sufficient to make educated decisions as the most important knowledge is hidden within the captured data.
+   
+Classification techniques are within the most utilised algorithms in order to extract hidden patterns. One major limitation of such classification models is the assumption that the underlying data concepts will not change over time. This assumption poses a major classification limitation as in the real-world observations and classes do change over time.  A real-world example is a spam filter where spammers continuously find new ways of how to send spam to increase their success rate.  Thus, a classification technique will decrease it accuracy over time as the observations, statistics and their classifications will not hold forever. This problem is referred to as concept drift. This increases machine learning model’s complexity, mostly in those cases where new data is treated as an important contributor to the final concept classification \citep{Bishop:2006:PRM:1162264} \citep{Tsymbal04theproblem}. 
```

Comment:
citations can be together \citep{cite1, cite2}

Reply:
done